src/core_ct/core.py

```diff
@@ -154,6 +155,116 @@ def trim_by_percent(self, axis: int, percent_start: float,
 
         return self.trim(axis, loc_start, loc_end)
 
+    def trim_radial(self,
+                    axis: int,
+                    radius: float, 
+                    x_center: int|None = None,
+                    y_center: int|None = None,
+                    z_center: int|None = None
```

Comment:
Feels a bit odd to have these all new lined like this when the other functions do not have them like this. Might just be me but would it be better to either have them all like this or none of them like this?